### DISCUSSION TITLE
Previewing result of JS blocks

### DISCUSSION THREAD

**Discussion by usernotnull** [chhoumann/quickadd#501] [Q&A] [answered]
When writing a command to AI, I may write something as below:

```js quickadd
const dv = DataviewAPI;

return dv.pages('"Indexes"').file.tags.join(', ')
```

Is there a way to preview what will be the result of the above block?

**Comment by chhoumann** [comment]
Best way is with the Dev Tools Console. Ctrl + Shift + i will show it. Cmd instead of Ctrl on Mac. 

Then you can paste the code into the console and run it. But you don't have access to the QuickAdd API from `this` there.

**Reply by usernotnull** [reply]
Thanks for the quick reply. The issue is I'm getting the error "Uncaught SyntaxError: Illegal return statement" if I copy the block above.

**Reply by chhoumann** [reply, marked as answer]
Yeah, since it's not inside a function, `return` isn't supported. You can just remove it
